galite-tests/src/test/kotlin/org/kopi/galite/tests/chart/ChartSample.kt

```diff
@@ -86,18 +88,18 @@ object ChartSample: Chart() {
 
   init {
     city.add("Tunis") {
-      this[area] = 34600
+      this[area] = decimal("34600")
       this[population] = 1056247
     }
 
     city.add("Kasserine") {
-      this[area] = 806600
+      this[area] = decimal("806600")
```

Comment:
what about : decimal(806600.0) to show that we can create decmal from an Double

Reply:
The recommanded way is to use string, floats are not precise